crates/sui-graphql-rpc/src/types/move_value.rs

```diff
@@ -123,12 +123,11 @@ impl MoveValue {
         // TODO: If this becomes a performance bottleneck, it can be made more efficient by not
         // deserializing via `value::MoveValue` (but this is significantly more code).
         bcs::from_bytes_seed(&layout, &self.bcs.0[..]).map_err(|_| {
-            let type_tag: Option<TypeTag> = (&layout).try_into().ok();
-            Error::Internal(if let Some(type_tag) = type_tag {
-                format!("Failed to deserialize Move value for type: {}", type_tag)
-            } else {
-                "Failed to deserialize Move value for type: <unknown>".to_string()
-            })
+            let type_tag: TypeTag = (&layout).into();
```

Comment:
Calling this out as a behavior change I believe. cc @amnn I think this change is probably fine, but you'd know better than me here :)

Reply:
LGTM, thanks for flagging!